app/MyApp/app/collections/CourseAnswer.js

```diff
@@ -3,7 +3,9 @@ $(function() {
     App.Collections.CourseAnswer = Backbone.Collection.extend({
 
         url: function() {
-            if (this.MemberID != "" && this.StepID != "" && this.pqattempts != "") {
+            if (this.MemberID != "" && this.StepID != "") {
+                return App.Server + '/courseanswer/_design/bell/_view/AnswerByMemberStepId/?key=["' +this.MemberID + '","' +this.StepID+ '"]&include_docs=true'
+            } else if (this.MemberID != "" && this.StepID != "" && this.pqattempts != "") {
```

Comment:
this condition will never be called as previous condition will always be true for this condition to meet